.github/workflows/tests.yml

```diff
@@ -7,8 +7,8 @@ on:
   pull_request:
 
 jobs:
-  lint-all-check:
-    name: yarn lint-all-check
+  test:
```

Comment:
Test coverage is good to add to the CI, but we don't want to remove lint checks. We should have separate jobs for those tasks.

Reply:
I've left the list coverage in the pre-existing, but I can move both tests into the same file (or workflow) if that's better.

Reply:
Oh my apologies, Teymour! I misunderstood what was happening here because 1) I was accidentally looking at only your second commit and thought this was differential instead of additive, and 2) it's early in my timezone. This looks great to me with clearer eyes. Thanks!

Reply:
It might be more efficient to run lints and tests as part of the same job, because `yarn install` only has to be run once.